### PR TITLE
Disable vacation messaging

### PR DESCRIPTION
**Pull request by BenHenning**
## Explanation

Now that the team is back for reviews, the automated PR message is no longer needed. This PR keeps the messaging support, but disables the vacation-specific message used over the last few weeks.

## Essential Checklist

- [ ] ~The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)~
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

N/A -- CI workflow change

### CODE REVIEW

**Comment by BenHenning** [comment]
@seanlip @rt4914 @anandwana001 PTAL. I'll merge this once I get one approval; just sending it broadly to increase chances of faster turnaround so that this message gets disabled.

File: .github/workflows/repository_messaging.yml
Comment (seanlip) [inline .github/workflows/repository_messaging.yml:13]: Perhaps worth adding a comment here saying something like "If situation XXX occurs, do YYY to turn this on." (It doesn't seem obvious to me as a possible future contributor that the natural thing to do is to drop this line altogether.)

**Review by seanlip — COMMENTED** [review]
Hi Ben -- left one suggestion, thanks!

**Review by anandwana001 — APPROVED** [review]
LGTM, agreed with Sean's comment about adding a comment to make true/false thing more understandable.

File: .github/workflows/repository_messaging.yml
Comment (BenHenning) [reply .github/workflows/repository_messaging.yml:13]: Good suggestion--done.

**Review by seanlip — APPROVED** [review]
Thanks, LGTM

**Comment by BenHenning** [comment]
Thanks @seanlip & @anandwana001. Enabling auto-merge.

**Pull request merged**
